src/main/java/org/broadinstitute/hellbender/utils/haplotype/PartiallyDeterminedHaplotype.java

```diff
@@ -162,23 +157,11 @@ public int hashCode() {
         return Objects.hash(Arrays.hashCode(getBases()),Arrays.hashCode(alternateBases));
     }
 
-    public List<Event> getDeterminedAlleles() {
-        return isDeterminedAlleleRef ? Collections.emptyList() : Collections.singletonList(alleleBearingEvent);
-    }
-
-    public void setAllDeterminedEventsAtThisSite(List<Event> allDeterminedEventsAtThisSite) {
-        this.allDeterminedEventsAtThisSite = allDeterminedEventsAtThisSite;
-        cachedExtent = null;
+    public Set<Event> getDeterminedEvents() {
+        return determinedEvents;
```

Comment:
This may or may not work when I get deeper into joint detection, but for now the branch solves the issue by allowing an empty set of determined events at the determined position (later, the determined interval).  This uniquely denotes that the reference is determined.